.serena/memories/project_structure.md

```
@@ -0,0 +1,98 @@
+# Project Structure
+
+## Root Directory
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)